CHANGELOG.md

```diff
@@ -29,13 +29,15 @@ Thank you to all who have contributed!
 ## [Unreleased]
 
 ### Added
+- **EXPERIMENTAL** Adds `EXCLUDE` to parser, ast, plan, and plan schema inferencer
```

Comment:
Could we also be explicit and make it clear that we are not supporting the evaluation at this stage.

Reply:
Sure, added a note to the CHANGELOG